VideoApp/VideoApp/API/Passcode/Core/API.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  Copyright (C) 2020 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
```

Comment:
I made this API client generic to be more single responsibility and reuse more code. This is the only file that imports Alamofire and it's really only doing Alamofire things. So it is a strong boundary between 3rd party code. To use just config with correct host and pass it an APIRequest. See FetchTwilioAccessTokenRequest for example.

I think we will be able to use this client instead of TwilioVideoAppAPI in the future by adding a little more configurability.

Reply:
I think in my next PR to fix environment setting I may try to replace old API code with new API client since I will have to touch that code anyway. Should be pretty easy.

Reply:
Exactly.